internal/provider/resource_group.go

```diff
@@ -118,9 +118,15 @@ func resourceGroupCreate(ctx context.Context, d *schema.ResourceData, meta inter
 		}
 		gcsmr, err := grps.SetMembers(ctx, gcr.Item.Id, gcr.Item.Version, memberIds)
 		if err != nil {
+			if err := setFromGroupResponseMap(d, raw); err != nil {
+				return diag.FromErr(err)
+			}
```

Comment:
What do you think about deferring this function call instead? Looks to me like we always want to call it after we've successfully called `grps.Create`.

Reply:
I was thinking that initially too, but @tmessi proposed that it might be better to explicitly call the set function when needed. I can go with either implementation.

Reply:
Not sure why having an explicit call is better, I would definitely prefer the defer.